src/app/screens/confirmStxTransaction/index.tsx

```diff
@@ -131,12 +133,25 @@ function ConfirmStxTransaction() {
     return microstacksToStx(amountToTransfer);
   };
 
-  const handleOnConfirmClick = (txs: StacksTransaction[]) => {
+  const handleConfirmClick = (txs: StacksTransaction[]) => {
+    if (isLedgerAccount(selectedAccount)) {
+      const type: LedgerTransactionType = 'STX';
+      navigate('/confirm-ledger-tx', {
+        state: {
+          unsignedTx: txs[0].serialize(),
+          type,
+          recipients: [{ address: recipient, amountSats: amount }],
```

Comment:
I think we should create a separate prop for STX transactions or the type for `recipients` should be updated since the type for `recipients` is currently set as `Recipient` and this is specific to BTC transactions only. https://github.com/secretkeylabs/xverse-core/blob/58d6c310a23cf36e35c1f0bd14a5de20a5ebe2f0/transactions/btc.ts#L25

 The `amountSats` field implies the amount is in satoshis but for STX transactions the amount is actually in microstacks

Reply:
Agree, I added the `StacksRecipient` type and started using it for `confirmStxTransaction/index.tsx`, `confirmNftTransaction/index.tsx`, and `confirmFtTransaction/index.tsx`